airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -386,14 +385,16 @@ def attempt_task_runs(self):
                     )
                     self.pending_tasks.append(ecs_task)
                 else:
-                    self.log.error(
```

Comment:
@syedahsn You should have a review of this if you get a chance. The failure reason handling has been modified here.